examples/java/src/main/java/org/apache/beam/examples/cookbook/BigQueryTornadoes.java

```diff
@@ -148,13 +151,23 @@ public void processElement(ProcessContext c) {
   static void runBigQueryTornadoes(Options options) {
     Pipeline p = Pipeline.create(options);
 
+    // Build the read options proto for the read operation.
+    TableReadOptions tableReadOptions =
+        TableReadOptions.newBuilder()
+            .addAllSelectedFields(Lists.newArrayList("month", "tornado"))
+            .build();
+
     // Build the table schema for the output table.
     List<TableFieldSchema> fields = new ArrayList<>();
     fields.add(new TableFieldSchema().setName("month").setType("INTEGER"));
     fields.add(new TableFieldSchema().setName("tornado_count").setType("INTEGER"));
     TableSchema schema = new TableSchema().setFields(fields);
 
-    p.apply(BigQueryIO.readTableRows().from(options.getInput()))
+    p.apply(
```

Comment:
Please add storage API as an option instead of changing the default.

Reply:
(Till we have dynamic work rebalancing support and ready to declare storage API as the default for production use.)

Reply:
Done.